app/templates/main/results.html

```diff
@@ -68,7 +97,7 @@ <h2>
             <br>
               <p style="font-size: 20px; font-weight: 600; text-transform: uppercase; color: #0f4d90">No Results Found</p>
             {% else %}
-            <div id="result-list-target"></div>
+            <div id="result-list-target" page_size="20"></div>
```

Comment:
so now you're able to pass in a page_size variable into the React component prop. The result list component will take the page_size and render that number of entries per page.